packages/libclang/libclang.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata minClientVersion="2.12">
+    <id>libclang</id>
+    <version>8.0.0</version>
+    <authors>Microsoft and Contributors</authors>
```

Comment:
NuGet lists this as being the package authors/owners, not the binary authors/owners.